rep-0144.rst

```diff
@@ -0,0 +1,129 @@
+REP: 144
+Title: ROS Package Naming and Versioning
+Author: Vincent Rabaud <vincent.rabaud@gmail.com>
+Status: Draft
+Type: Informational
+Content-Type: text/x-rst
+Created: 28-Jan-2015
+Post-History:
+
+
+Abstract
+========
+
+This REP gives advice on how a ROS package should be named and versioned. It
+formalizes and extends conventions that were formerly described in [1]_.
+
+Motivation
+==========
+
+As the number of ROS packages increases, it is hard to quickly find a package
+and guess its functionality based on its name. Over time, the lack of naming 
+conventions created problems like use of unexplained acronyms or
+packages with the same functionality but different names.
+
+This REP proposes rules to name a ROS package properly. Those 
+rules are not mandatory, merely informative.
```

Comment:
It might make sense to actually split the mandatory rules into a separate list within the REP, for instance the alphanumerics and _ are mandatory, the avoidance of "ros" is not.